llvm/test/Transforms/DeadStoreElimination/inter-procedural.ll

```diff
@@ -0,0 +1,175 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
Please actually generate the test checks...

Reply:
I copied this note from other unit tests. From my understanding, `CHECK-NEXT` in this `ll` file is the assertion. Did I miss something?

Reply:
Got it, thanks for this info! Ran the script and updated all CHECK lines.